fbcnms-packages/fbcnms-ui/master/OrganizationEdit.js

```diff
@@ -175,205 +170,120 @@ export default function OrganizationEdit(props: Props) {
-                margin="normal"
-              />
-            </FormGroup>
-            <FormGroup>
-              <FormControl className={classes.textField} margin="normal">
-                <InputLabel htmlFor="tabs">Accessible Tabs</InputLabel>
-                <Select
-                  multiple
-                  value={Array.from(tabs)}
-                  onChange={({target}) => setTabs(new Set(target.value))}
-                  renderValue={value => {
-                    const selectedTabs = new Set(value);
-                    return renderList(
-                      allTabs
-                        .filter(tab => selectedTabs.has(tab.id))
-                        .map(x => x.name),
-                    );
+    <>
+      <div className={classes.container}>
+        <OrganizationDialog
+          hideAdvancedFields={props.hideAdvancedFields}
+          edit={true}
+          organization={organization}
+          open={dialog}
+          addUser={addUser}
+          createError={createError}
+          setAddUser={() => setAddUser(true)}
+          onClose={() => {
+            setAddUser(false);
+            setDialog(false);
+          }}
+          onCreateOrg={org => {
+            onSave(org);
+            setRefresh(Date.now());
+            setDialog(false);
+          }}
+          onCreateUser={user => {
+            axios
+              .post(
+                `/master/organization/async/${
+                  match.params.name || ''
+                }/add_user`,
+                user,
+              )
+              .then(() => {
+                enqueueSnackbar('User added successfully', {
+                  variant: 'success',
+                });
+                setDialog(false);
+                history.push(relativeUrl(''));
+              })
+              .catch(error => {
+                setCreateError(error);
+                enqueueSnackbar(error?.response?.data?.error || error, {
+                  variant: 'error',
+                });
+              });
+          }}
+        />
+        <Grid container spacing={4}>
+          <Grid container justify="space-between">
+            <Grid item>
+              <Grid container alignItems="center">
+                <Grid>
+                  <IconButton
+                    onClick={() => history.goBack()}
+                    className={classes.arrowBack}
+                    color="primary">
+                    <ArrowBackIcon />
+                  </IconButton>
+                </Grid>
+                <Grid>
+                  <Text className={classes.header} variant="h4">
+                    {organization.name}
+                  </Text>
+                </Grid>
+              </Grid>
+            </Grid>
+            <Grid item>
+              <Button
+                skin="comet"
+                onClick={() => {
+                  props
+                    .confirm('Are you sure you want to delete this org?')
```

Comment:
Avoid abbreviations if possible.
```suggestion
                    .confirm('Are you sure you want to delete this organization?')
```